datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -608,6 +609,8 @@ protected void maybeCommitOffsetsInternal(Consumer<?, ?> consumer, boolean force
 
   protected void commitWithRetries(Consumer<?, ?> consumer, Optional<Map<TopicPartition, OffsetAndMetadata>> offsets)
       throws DatastreamRuntimeException {
+    preCommitHook();
+    AtomicReference<KafkaException> lastKafkaExceptionRef = new AtomicReference<>(null);
```

Comment:
Can we remove this now that we no longer need to pass it to the postCommitHook()?

Reply:
Done